thirdparty/runtime_environments_test.go

```diff
@@ -21,3 +22,38 @@ func TestGetImageNames(t *testing.T) {
 	assert.NotEmpty(result)
 	assert.NotContains(result, "")
 }
+
+func TestGetPackages(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	assert := assert.New(t)
+	config := testutil.TestConfig()
+	testutil.ConfigureIntegrationTest(t, config, "TestGetPackages")
+	c := NewRuntimeEnvironmentsClient(config.RuntimeEnvironments.BaseURL, config.RuntimeEnvironments.APIKey)
+	manager := "pip"
+	ami := "ami-0e12ef25a5f7712a4"
+	opts1 := PackageFilterOptions{
+		Page:    0,
+		Limit:   10,
+		Manager: manager,
+	}
```

Comment:
you can just call this `opts` and overwrite it in later executions (see next comment)
```suggestion
	opts := PackageFilterOptions{
		Page:    0,
		Limit:   10,
		Manager: manager,
	}
```